swatch-contracts/src/main/resources/db/202310091227_fix_contract_metrics_for_rosa.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+    https://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd">
+
+<changeSet id="202310091227-1" author="ksynvrit">
+    <comment>Change four_vcpu_hour to Cores on contract_metrics.</comment>
+    <update tableName="contract_metrics">
+        <column name="metric_id" value="Cores"/>
```

Comment:
We shouldn't change the values, in my opinion, since these values are of the aws dimension and not the actual metric id. The aws dimension should stay in the contract_metrics table and shouldn't be replaced. This problem has to be solved in different way because `contractEntityToSubscriptionProductIdEntity` in ContractMapper maps to incorrect values due to same name `metric_id`. So either we need to rename the column to cloud_dimension or `contractEntityToSubscriptionProductIdEntity` should have the right mapping.